ride_share.rb

```diff
@@ -0,0 +1,61 @@
+rides = {
+  DR0004: {
+    dates: ["3rd Feb 2016","4th Feb 2016", "5th Feb 2016"],
+    costs: [5, 10, 20],
+    rider_ids: ["RD0022", "RD0022", "RD0073"],
+    ratings: [5, 4, 5]
+    },
+  DR0003: { 
+    dates: ["4th Feb 2016", "5th Feb 2016"],
+    costs: [5, 50],
+    rider_ids: ["RD0066", "RD0003"],
+    ratings: [5, 2]
+    },
+  DR0002: {
+    dates: ["3rd Feb 2016", "4th Feb 2016", "5th Feb 2016"],
+    costs: [25, 15, 35],
+    rider_ids: ["RD0073", "RD0013", "RD0066"],
+    ratings: [5, 1, 3]
+    }, 
+  DR0001:{
+    dates: ["3rd Feb 2016", "3rd Feb 2016", "5th Feb 2016"],
+    costs: [10, 30, 45],
+    rider_ids: ["RD0003", "RD0015", "RD0003"],
+    ratings: [3, 4, 2]
+    } 
+}
+
+def stats(hash, driver_id)
+  amount_made = 0
+  sum_rating = 0
+  number_rides = hash[:dates].length 
+  
+  hash[:costs].each do |i|
+    amount_made = amount_made + i
+  end
+  
+  hash[:ratings].each do |i|
+      sum_rating = sum_rating + i
```

Comment:
You over-indented this line.

```suggestion
    sum_rating = sum_rating + i
```